File: Python/URI/Beginner/1045.py
from math import pow

a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

v = [a,b,c]
v.sort()

a = v[2]
b = v[1]
c = v[0]

if ((a >= (b + c))):
	print('NAO FORMA TRIANGULO')
else:
	if ((pow(a,2) == pow(b,2) + pow(c,2))):
		print('TRIANGULO RETANGULO')
	if ((pow(a,2) > pow(b,2) + pow(c,2))):
		print('TRIANGULO OBTUSANGULO')
	if ((pow(a,2) < pow(b,2) + pow(c,2))):
		print('TRIANGULO ACUTANGULO')
	if (a == b and b == c):
		print('TRIANGULO EQUILATERO')
	if (((a == b) and (b != c)) or ((b == c) and (a != c)) or ((a==c) and (a !=c))):
		print('TRIANGULO ISOSCELES')

File: URI/Beginner/1045.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {

    vector<double> v;
    double n;

    for (int i=0;i<3;i++){
        cin >> n;
        v.push_back(n);
    }

    sort(v.begin(), v.begin() + 3);

    double a = v[2], b = v[1], c = v[0];

    if (a >= (b + c)) {
        printf("NAO FORMA TRIANGULO\n");
        return 0;
    }
    if (a*a == (b*b + c*c)) printf("TRIANGULO RETANGULO\n");
    if (a*a > (b*b + c*c)) printf("TRIANGULO OBTUSANGULO\n");
    if (a*a < (b*b + c*c)) printf("TRIANGULO ACUTANGULO\n");
    if (a == b && b == c) printf("TRIANGULO EQUILATERO\n");
    if ((a == b && b != c) || (a == c && c != b) || (b == c && c != a)) printf("TRIANGULO ISOSCELES\n");


    return 0;
}